core/src/main/java/jenkins/util/TreeStringBuilder.java

```diff
@@ -20,8 +20,6 @@
  * @author Kohsuke Kawaguchi
  * @since 1.473
  */
-@SuppressWarnings({"PMD", "all"})
-//CHECKSTYLE:OFF
```

Comment:
This file complies with our Checkstyle checks, so I see no reason to turn Checkstyle off.